tests/src/test/java/tachyon/shell/command/ChownCommandTest.java

```diff
@@ -42,4 +42,18 @@ public void chownTest() throws IOException, TachyonException {
     owner = mFileSystem.getStatus(new TachyonURI("/testFile")).getUserName();
     Assert.assertEquals("user2", owner);
   }
+
+  @Test
```

Comment:
Please add a brief javadoc on the testcaes method, for testing javadoc you don't need to explain exception.